resources/harbor/exporter-config.yaml

```diff
@@ -0,0 +1,156 @@
+---
+apiVersion: v1
+kind: ExporterConfig
+app: "Harbor"
+version: 1.0.0
+appVersion: 
+  - 1.10
+maintainers:
+  - name: Sysdig team
+    link: https://sysdig.com/
+description: |
```

Comment:
How about a link to the blogpost here?

Reply:
Have we a blog created already?

Reply:
um, maybe is not published? you can arrange the url with Golo so it doesn't change